### ISSUE TITLE
CASE überdenken

### ISSUE THREAD

**Issue by SonjaKaufmann** [Optinomic/apps#147]
Wir sollten überlegen, wie wir den CASE einbinden. Eventuell braucht es eine neue App, denn der Summenwert sagt nichts aus.
Sonja muss dies mit der Therapieleitung anschauen.

**Comment by schoenenb** [comment]
Gedanken zum CASE auch hier:
https://github.com/Optinomic/apps/issues/135#issuecomment-338574596 & https://github.com/Optinomic/apps/issues/135#issuecomment-338665932

> Stimmt. Ich finde der CASE Eintritt sowie Austritt enthält wichtige Infos und finde es schade das er in einem einzigen Summenwert dargestellt wird. Da sollten wir nach der Einführung nochmal drüber nachdenken. Werde dann auch noch das Gespräch mit der Therapieleitung suchen, aber eigentlich möchten sie ja gerne auf einen Blick wissen, wieviel Entzüge z.B. schon stattgefunden haben und so. Wäre wohl sinnvoll mal die alten CASE-Daten auszuwerten, was davon zur Prognose des Therapieerfolges sinnvolle Informationen bietet. Aber ich denke das vertagen wir für nach der Einführung.

Eine kleine Anmerkung noch: meines Wissens wird der CASE nur beim Eintritt erhoben, hat sich da was geändert? Oder meinst du damit auch die Daten vom Einzelassessment/Neuroanamnese?

**Comment by SonjaKaufmann** [comment]
Eine Therapeutin hat gemeldet, dass sie den CASE nicht mehr auswählen kann und somit auch nicht eingeben. Ist der nicht im Neuroassessment drin? Konnte ihn auch nicht separat finden. Wo wird der eingegeben?

**Comment by schoenenb** [comment]
Nein, der Fragebogen ist separat. Der parent wurde umbenannt (von case.new in case), weshalb der Pfad nicht mehr auffindbar war. Der CASE ist nun wieder als App verfügbar und wird bei allen, wo er bereits aktiviert war, wieder angezeigt.
